airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -127,6 +127,8 @@ def read_records(self, stream_slice: Mapping[str, any] = None, **kwargs) -> Iter
                 # For private repositories `Teams` stream is not available and we get "404 Client Error: Not Found for
                 # url: https://api.github.com/orgs/sherifnada/teams?per_page=100" error.
                 error_msg = f"Syncing `Team` stream isn't available for repository `{stream_slice['repository']}`."
+            else:
+                raise e
```

Comment:
Can we have an additional information where this error is being thrown? Maybe 
```python
else:
    logging.ERROR("Undefined error while reading records")
    raise e
```

Reply:
Done